### PR TITLE
added consul acl token note

### PR DESCRIPTION
**Pull request by bitsofinfo**
Documentation update on how to specify a consul acl token: https://github.com/containous/traefik/issues/1717

### CODE REVIEW

File: docs/user-guide/kv-config.md [hunk 1]
```diff
@@ -137,17 +137,24 @@ traefik:
 
 NB : Be careful to give the correct IP address and port in the flag `--consul.endpoint`.
 
+## Consul ACL Token support
+
+To specify a Consul ACL token for traefik to use you must set an system environment variable
+prior to launching traefik named `CONSUL_HTTP_TOKEN` equal to the acl token value.
```
Comment (nmengin) [inline docs/user-guide/kv-config.md]: Maybe the sentence can looks like this : `To specify a Consul ACL token for Traefik, we have to set a System Environment variable named **CONSUL_HTTP_TOKEN** prior to starting traefik. This variable must be initialized with the ACL token value.`

File: docs/user-guide/kv-config.md [hunk 2]
```diff
@@ -137,17 +137,24 @@ traefik:
...
+
+If running traefik as a docker container you can do it like: `-e "CONSUL_HTTP_TOKEN=[consul-acl-token-value]"`
```
Comment (nmengin) [inline docs/user-guide/kv-config.md]: Maybe the sentence can looks like this : `If Traefik is launched into a Docker container, the variable **CONSUL_HTTP_TOKEN** can be initialized with the `-e` Docker option : `-e "CONSUL_HTTP_TOKEN=[consul-acl-token-value]"``

File: docs/user-guide/kv-config.md (hunk 1 above)
Comment (bitsofinfo) [inline docs/user-guide/kv-config.md]: sure, done

File: docs/user-guide/kv-config.md (hunk 2 above)
Comment (bitsofinfo) [inline docs/user-guide/kv-config.md]: sure, done

**Pull request merged**